### ISSUE TITLE
Issue with PHP7

### ISSUE THREAD

**Issue by saqibsarwar** [lesterchan/wp-serverinfo#2]
Hi,

I installed this plugin on Site Ground and my website started having 500 server error due to some conflict with PHP7. I removed the plugin and error got fixed.

Kindly look into fixing this plugin for PHP7.

Thanks & Regards

**Comment by lesterchan** [comment]
Do you have the exact error? I don't use PHP7.0 myself so I can't test it

I added PHP 7.0 synatx check and there are no errors https://travis-ci.org/lesterchan/wp-serverinfo/jobs/106125512

**Comment by saqibsarwar** [comment]
Thanks for responding.

Unfortunately I do not have the exact issue. But you can change PHP version by following something like https://www.siteground.com/kb/how_to_have_different_php__mysql_versions/

If you like I can give you access to my testing server and for that you can get in touch with me at saqibsarwar at gmail.

**Comment by lesterchan** [comment]
I might not have time for it, so I will leave this open =)

**Comment by ihorvorotnov** [comment]
I'm using it on PHP 7 without any issues, except it does not recognize memcached (but that's another story).

**Comment by lesterchan** [comment]
To add on, recently I have problem with HHVM and I switched to PHP7 and it works fine. I have two memcache and memcached extensions.  This plugin detects the Memcache (without the d).

**Comment by ihorvorotnov** [comment]
Yes, I've checked the code already. Any plans to support memcache**d**? I'd love to fork the plugin, add it by myself and send a PR but have no clue when it can happen - totally busy for next couple months.

**Comment by edwarddekker** [comment]
Hi,

At line 381 @ wp-serverinfo.php do I see a problem for PHP7.

### Function: Get PHP Safe Mode
 if(!function_exists('get_php_safe_mode')) {
    function get_php_safe_mode() {
        if(ini_get('safe_mode')) {
            $safe_mode = __('On', 'wp-serverinfo');
        } else {
            $safe_mode = __('Off', 'wp-serverinfo');
        }
        return $safe_mode;
    }
}

This is a depreciated code since PHP5.4

**Comment by lesterchan** [comment]
@ihorvorotnov Feel free to forked it and send a PR =)
